x-pack/filebeat/input/s3/config.go

```diff
@@ -18,6 +18,7 @@ type config struct {
 	VisibilityTimeout         time.Duration       `config:"visibility_timeout"`
 	AwsConfig                 awscommon.ConfigAWS `config:",inline"`
 	ExpandEventListFromField  string              `config:"expand_event_list_from_field"`
+	ContextTimeout            time.Duration       `config:"context_timeout"`
```

Comment:
context is probably not very descriptive, how about s3_read_timeout?

Reply:
Good point, changed to aws_api_timeout for all AWS api requests.